source/contracts/Apple.sol

```diff
@@ -1,6 +1,6 @@
 pragma solidity ^0.4.13;
 
-import 'ROOT/Controlled.sol';
```

Comment:
I have been thinking about this a lot lately.  Solidity supports any string as the "global name" for a file.  What this means is that whatever string we use when adding the file to the standard-json input to solidity is the string that we need to use to import he file.  This means we could do something like `trading/apple.sol` or just `apple.sol` or `./trading/apple.sol`.  It also supports remappings, which is something I did originally before I really understood how the system worked.

I think remappings like we had before (`ROOT`) is the worst option.  What you have here is definitely better, though I wonder if we could save ourselves some effort by not requiring the leading `./`?  In this case, the import would be simplified to `Controlled.sol`.  This would require a minor change in the script that builds the standard-json, but that should be relatively minor.

Reply:
Ok, so would this have to be done using the `remappings` JSON parameter in my script or a different parameter?  I see in conftest.py we have:

```         
compilerParameter = {
            'language': 'Solidity',
            'sources': {
                absoluteFilePath: {
                    'urls': [ absoluteFilePath ]
                }
            },
            'settings': {
                'remappings': [ 'ROOT=%s' % resolveRelativePath("../src") ],
                'outputSelection': {
                    '*': [ 'metadata', 'evm.bytecode', 'evm.sourceMap' ]
                }
            }
        }
```
Also, I just realized I'm outputting only the bytecode to the output file, but I see here we are outputting:
```
[ 'metadata', 'evm.bytecode', 'evm.sourceMap' ]
```
I assume I'll need to change this in my script?

Reply:
I don't think we shold use remappings anymore.  I did that because I didn't understand how file lookups in solc worked and it fixed my problem at the time.  Now that I understand better, I think it isn't the right choice.

Lets go with just the default outputs for now.  I don't know what they are, but I don't think it is bytecode only by default?

Reply:
No, I think by default it outputs everything.  The fields I see are: `assembly`, `bytecode`, `functionHashes`, `gasEstimates`, `interface`, `metadata`, `opcodes`, `runtimeBytecode`, `srcmap`, & `srcmapRuntime`.